packages/commands/src/accounts/create.ts

```diff
@@ -108,7 +108,7 @@ export const createAccountsCommand = new Command("create")
       AvailableAccountTypes
     )
   )
-  .requiredOption("--name <name>", "Account name")
+  .option("--name <name>", "Account name", "testnet")
```

Comment:
I like the idea of having a default account name if none is provided, but calling it `"testnet"` can be confusing, especially considering it could be used to do things in other environments (like fetching balances on mainnet, doing things on localnet, etc). We should probably go for something more generic like "user", "default" or "account1"

Reply:
Users really shouldn't be mixing accounts. Localnet will likely automatically be creating an account named `localnet`. Since all our commands default to testnet, I thought having the default named testnet made sense, but I suppose we can keep it called "default" for now.

Reply:
We should probably have the "default" account name stored as a constant and imported where applicable, it might sound irrelevant, but it will be used in a bunch of different places, let's avoid hardcoding it everywhere.